sql/updates/world/master/2021_02_17_00_world_druid_gore.sql

```diff
@@ -0,0 +1,9 @@
+-- Attach druid's gore spell script
+DELETE FROM `spell_script_names` WHERE `ScriptName` = 'spell_dru_gore';
+INSERT INTO `spell_script_names` (`spell_id`,`ScriptName`) VALUES
+(210706, 'spell_dru_gore');
+
+-- Override Gore spell proc flags
+DELETE FROM `spell_proc` WHERE `SpellId` = 210706;
+INSERT INTO `spell_proc` (`SpellId`, `SchoolMask`, `SpellFamilyName`, `SpellFamilyMask0`, `SpellFamilyMask1`, `SpellFamilyMask2`, `SpellFamilyMask3`, `ProcFlags`, `SpellTypeMask`, `SpellPhaseMask`, `HitMask`, `AttributesMask`, `DisableEffectsMask`, `ProcsPerMinute`, `Chance` , `Cooldown`, `Charges`) VALUES
+(210706, 65, 7, 0x00200800, 0x00100000, 0x08000000, 0, 0x00011010, 1, 1, 0x403, 0, 0, 0, 15, 0, 0);
```

Comment:
Leave 0 in fields you do not want to override (Chance)

Reply:
https://github.com/TrinityCore/TrinityCore/blob/master/src/server/game/Spells/SpellMgr.cpp#L1545
SchoolMask is always overriden?

I'll use 0 for the 2 others though.

Reply:
https://github.com/TrinityCore/TrinityCore/blob/456b5be8e45f5718287a44c1a6b8ba2429051f01/src/server/game/Spells/SpellMgr.cpp#L524

Reply:
For other people reading this:

Any SchoolMask value used in spell_proc will override the default. However, using value of 0 will disable that check which is preferred